tests/figC.ipynb

```diff
@@ -0,0 +1,416 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pio7uVdRqIfg"
+   },
+   "source": [
+    "#### Get the lineage tree data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To group the wastewater data properly for graphing, we need to get lineage data from the pre-parsed lineage tree (derived from the Pangolin project). This will help us organize and cluster lineages later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {
+    "id": "UiDTSwvsqLAW"
+   },
+   "outputs": [],
+   "source": [
+    "tree = outbreak_clustering.get_compressed_tree()\n",
+    "lineage_key = outbreak_clustering.get_lineage_key(tree)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 56,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "samples_per_site = ww_samples['sra_accession'].groupby(ww_samples['collection_site_id']).nunique()\n",
+    "\n",
+    "date_range_per_site = ww_samples.index.to_series().apply(pd.to_datetime).groupby(ww_samples['collection_site_id']) #.apply(pd.to_datetime)\n",
+    "date_range_per_site = date_range_per_site.apply(lambda x: x.max() - x.min())\n",
+    "\n",
+    "sites_for_vl = (date_range_per_site.apply(lambda x: x.days >= 14)) & (samples_per_site >= 3)\n",
+    "samples_for_vl = ww_samples[ww_samples['collection_site_id'].isin(list(sites_for_vl.index[sites_for_vl]))]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5AwfrVK9qRf1"
+   },
+   "source": [
+    "#### Cluster lineages "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we'll need to fetch and aggregate the viral load sample data to get our wastewater prevalence data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {
+    "id": "RQZqFobuqPyJ"
+   },
+   "outputs": [],
+   "source": [
+    "viral_load = samples_for_vl.set_index(pd.Series(['viral_load'] * len(samples_for_vl)), append=True)\n",
+    "\n",
+    "c = 'normed_viral_load'\n",
+    "viral_load_weekly, _ = outbreak_tools.datebin_and_agg(viral_load, weights=viral_load['ww_population'].fillna(1000), startdate=startdate, enddate=enddate, freq='7D', column=c, norm=False, rolling=[1,4,1], log=False, variance=True)\n",
+    "viral_load_weekly = viral_load_weekly['viral_load']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we generate clustered groups given our prevalence and lineage tree data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {
+    "id": "uk8EX6wSqXC6"
+   },
+   "outputs": [],
+   "source": [
+    "ww_prev_data = ww_prevalences.mul(viral_load_weekly, axis=0).sum()\n",
+    "clinical_prev_data = clinical_prevalences.mul(viral_load_weekly, axis=0).sum()\n",
+    "\n",
+    "ww_clusters = outbreak_clustering.cluster_lineages(ww_prev_data, tree, lineage_key=lineage_key, n=10, alpha=0.25)\n",
```

Comment:
Similarly to figAB, let's simplify by getting rid of the viral load info from this notebook and just clustering on ww_prevalences.sum(). We can use one set of clusters for both ww and clinical